standalone/instructlab-pipelines-with-dsp-on-rhoai-.md

```diff
@@ -0,0 +1,646 @@
+    name: kserve-container
+    ports:
+    - containerPort: 8080
+      protocol: TCP
+    volumeMounts:
+    - mountPath: /dev/shm
+      name: shm
+    - mountPath: /mnt
+      name: mixtral-serve
+  multiModel: false
+  storageHelper:
+    disabled: true
+  supportedModelFormats:
+  - autoSelect: true
+    name: vLLM
+  volumes:
+  - name: mixtral-serve
+    persistentVolumeClaim:
+      claimName: mixtral-serving-ilab
+  - emptyDir:
+      medium: Memory
+      sizeLimit: 2Gi
+    name: shm
+```
+
+</details>
+
+<details>
+
+<summary>inferenceservice.mixtral.yaml</summary>
+
+```yaml
+---
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  annotations:
+    openshift.io/display-name: mixtral
+    security.opendatahub.io/enable-auth: "true"
+    serving.knative.openshift.io/enablePassthrough: "true"
+    sidecar.istio.io/inject: "true"
+    sidecar.istio.io/rewriteAppHTTPProbers: "true"
+  finalizers:
+  - inferenceservice.finalizers
+  labels:
+    opendatahub.io/dashboard: "true"
+  name: mixtral
+spec:
+  predictor:
+    maxReplicas: 1
+    minReplicas: 1
+    model:
+      args:
+      - --dtype=bfloat16
+      - --tensor-parallel-size=4
+      - --enable-lora
+      - --max-lora-rank=64
+      - --lora-dtype=bfloat16
+      - --fully-sharded-loras
+      - --lora-modules
+      - skill-classifier-v3-clm=/mnt/models/skills
+      - text-classifier-knowledge-v3-clm=/mnt/models/knowledge
+      modelFormat:
+        name: vLLM
+      name: ""
+      resources:
+        limits:
+          cpu: "4"
+          memory: 60Gi
+          nvidia.com/gpu: "4"
+        requests:
+          cpu: "4"
+          memory: 60Gi
+          nvidia.com/gpu: "4"
+      runtime: mixtral
+      storage:
+        # the secret name of the secret deployed earlier
+        key: aws-connection-my-bucket
+        # update this to match the path in your bucket
+        path: <prefix-path-to-mixtral-model-in-s3>
+    tolerations:
+    - effect: NoSchedule
+      key: nvidia.com/gpu
+      operator: Exists
+```
+
+</details>
+
+```bash
+oc -n <data-science-project-name/namespace> apply -f servingruntime.mixtral.yaml
+oc -n <data-science-project-name/namespace> apply -f inferenceservice.mixtral.yaml
+```
+
+A new pod named `mixtral-predictor-0000#-deployment-<hash>` should be created. This should result in a successful
+running pod. If the pod does not come up successfully, you inspect the `.status` field for the `InferenceService`
+for issues.
+
+```bash
+oc -n <data-science-project-name/namespace> get inferenceservice mixtral -o yaml
+```
+
+#### Deploy teacher model serving details
+
+Create a secret containing the Teacher model serving details
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <teacher-model-details-k8s-secret>
+  namespace: <data-science-project-name/namespace>
+type: Opaque
+stringData:
+  api_key:              # Deployed model-server auth token
```

Comment:
same as above - lets provide `<placeholder-values>` to stay consistent in our yamls